usb/hid.c

```diff
@@ -345,7 +355,7 @@ static uint8_t usb_hid_init(usb_device_t *dev, usb_device_descriptor_t *dev_desc
 				return rcode;
 			}
 
-			if(info->iface[i].device_type == REPORT_TYPE_NONE) {
+			if(info->iface[i].device_type == HID_DEVICE_UNKNOWN) {
```

Comment:
device_type should compare against HID_DEVICE_*.
It worked anyways before because the same values were used for REPORT_TYPE_*.